pyproject.toml

```diff
@@ -0,0 +1,46 @@
+[build-system]
+requires = ["setuptools>=72", "wheel", "tomli"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "tfx-dev"
+dynamic = ["version", "dependencies", "optional-dependencies", "scripts"]
+description = "TensorFlow Extended (TFX) is a TensorFlow-based general-purpose machine learning platform implemented at Google."
+readme = "README.md"
+license = { file = "LICENSE" }
+authors = [
+    { name = "Google LLC", email = "tensorflow-extended-dev@googlegroups.com" }
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Education",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3 :: Only",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Artificial Intelligence",
+    "Topic :: Scientific/Engineering :: Mathematics",
+    "Topic :: Software Development",
+    "Topic :: Software Development :: Libraries",
+    "Topic :: Software Development :: Libraries :: Python Modules"
+]
+keywords = ["tensorflow", "tfx"]
+requires-python = ">=3.9,<3.11"
+[project.urls]
+Homepage = "https://www.tensorflow.org/tfx"
+Repository = "https://github.com/tensorflow/tfx"
+
+[tool.pytest.ini_options]
+addopts = "--verbose -m 'not end_to_end'"
```

Comment:
I'd vote to remove this line because
- Users should be in control of verbosity
- IMO all tests should run by default

Thoughts?

Reply:
Completely agree, but I would propose to handle it in a separate pull request to isolate and minimize functional changes, this is basically a copy of already existing `setup.cfg` (removed in this PR): https://github.com/tensorflow/tfx/blob/4012ecbe64d737ada5ae68ac8f8ce874bc23aa7a/setup.cfg#L5